src/routes/styles.css

```
@@ -7,6 +7,7 @@
   --lighterSteelBlue: rgb(102, 149, 180);
   --steelBlue: rgb(70, 130, 180);
   --notehubBlue: rgb(27, 58, 82);
+  --notehubBlueTintLighter: #729bb5;
```

For consistency using RGB everywhere

```suggestion
  --notehubBlueTintLighter: rgb(114, 155, 181);
```